tests/test_pfx_table.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf8 -*-
+"""
+tests.pfx_table_test
+------------------
+"""
+
+import os, sys
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+
+import unittest
+
+from rtrlib import PfxTable
+
+# flag constants for asserting the validation result
+VALID = 1
+INVALID = 2
+UNKNOWN = 4
+LENGTH_INVALID = 8
+AS_INVALID = 16
```

Comment:
I think it would be clearer like this:
```suggestion
VALID = 1 << 0
INVALID = 1 << 1
UNKNOWN = 1 << 2
LENGTH_INVALID = 1 << 3
AS_INVALID = 1 << 4
```